internal/guard/guard.go

```
@@ -0,0 +1,301 @@
+		CopyFrom:   src,
+		CopyTo:     absPath,
+		UnmountCmd: append([]string{"sudo"}, snapshot.UnmountArgs(mnt)...),
+	}
+	if print {
+		return plan, nil, nil
+	}
+
+	if err := os.MkdirAll(mnt, 0o755); err != nil {
+		return plan, nil, err
+	}
+	defer os.RemoveAll(mnt)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)